package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-oauth2",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
This is not needed. The version changes with `npm version major`

Reply:
Indeed, I've reverted this.